.gitignore

```diff
@@ -21,3 +21,6 @@
 /yarn-error.log
 
 .byebug_history
+
+/data
```

Comment:
Let's revisit this later

Reply:
Great catch with regard to cloning down the repo and using the data.